creating_packages/package_dev_flow.rst

```diff
@@ -148,8 +148,19 @@ This parameters takes the same parameters as ``package()`` but in this case ther
 | ``install-folder`` |                    |
 |                    |                    |
 | ``build-folder``   |                    |
+|                    |                    |
+| ``package-folder`` |                    |
 +--------------------+--------------------+
 
+There are 2 modes of operation:
+
+- Using ``source-folder`` and ``build-folder`` does not require the execution of the above ``$ conan package``.
```

Comment:
Why state that the ``$ conan package`` above is not needed? I mean, I know you don't but I think it is better to encourage users to execute it to make sure an to stick to a the same flow. I'd rather omit this *skip* of ``package()``

Reply:
It is important that users understand that ``export-pkg`` in this case works from the source and build folder, and not from the package folder, and that it will execute the ``package()`` method. Otherwise there can be confusion about what the command is actually doing.

Reply:
> does not require the execution of the above ``$ conan package``

It is OK with me, I would just omit the above sentence and keep the rest of the explanation